src/trufnetwork_sdk_py/client.py

```
@@ -3371,6 +3432,236 @@ def _validate_binary_market_inputs(
+                raise TypeError(f"body_hash must be bytes, hex string, or None, got {type(bh).__name__}")
+        if len(out) != n:
+            raise ValueError("body_hashes length must match namespaces length")
+        return out
+
+    @staticmethod
+    def _maa_rows(json_str: str) -> list[dict]:
+        """Parse the {column_names, values} JSON a read binding returns into a list of row dicts."""
+        data = json.loads(json_str)
+        cols = data.get("column_names") or []
+        vals = data.get("values") or []
+        return [dict(zip(cols, row)) for row in vals]
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)